README.md

```diff
@@ -167,10 +166,15 @@ A recent update  reduces the voltage supply from 3.3V to 2.7V  during sleep, the
 
 which with light use should give two or three days battery life for V1 watches and three or four days with V2 watches. (The V1 watch has no way to reset the touch driver chip and so it cannot be put into its power saving hibernate mode during light sleep)
 
-
-
-
-
-
-
- 
+### How to firmware flash
+
+* Install the latest **[thonny](https://thonny.org/)**
```

Comment:
You may want to consider adding instructions for using `esptool` in standalone as an alternative to Thonny for flashing the firmware: https://docs.micropython.org/en/v1.20.0/esp32/tutorial/intro.html#deploying-the-firmware

Reply:
Oh neat! I will add that on monday.

Reply:
I added the example.